### PR TITLE
Added analysis_plan.md

### PR DESCRIPTION
**Pull request by C-Muh**
@nipraxis-summer-2023/instructors - this plan is now ready for review.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -19,7 +19,7 @@ tar xvf group_data.tar
 Add the hash_list file to Git:
 
 ```
-git add data/group-*/hash_list.txt
+git add group-*/hash_list.txt
```
Comment (matthew-brett) [inline README.md:26]: It takes practice, but try to make sure that you keep the pull request clean of any distracting changes.

**Review by matthew-brett — COMMENTED** [review]
Yes - though - I was hoping for more development of your plan, thinking about your exact metrics, how you were going to tell whether the scans really were outliers or not, any reading you were planning to do, and so on.

**Comment by matthew-brett** [comment]
@oesteban - any thoughts?

File: analysis_plan.md [hunk 2]
```diff
@@ -0,0 +1,9 @@
+Project OUTLIERS
+
+(1) Adapt DVARS from exercices in findoutlie/metrics.py
+(2) Fill in findoutlie/detectors.py for IQR
+(3) Fill in findoutlie/outfind.py 
+(4) Test on one subject
+(5) Test on the 9 other subjects 
+(6) Add other metrics from MRIqc : framewise displacement, tSNR
```
Comment (oesteban) [inline analysis_plan.md]: Point number 6 has two issues:
- If you are not going to run MRIQC, but rather implement these metrics then it's better to remove MRIQC from the plan and think about what will be necessary to calculate these.
- If you are going to run MRIQC, then why using just these two?

```suggestion
(6) Add other metrics:
    (6.1) framewise displacement -> requires head motion estimation
    (6.2) tSNR -> requires (at the very least) brain extraction, but it is better if you can calculate it in ROIs more targetted.
```

File: analysis_plan.md [hunk 3]
```diff
@@ -0,0 +1,9 @@
...
+(7) Add motion detection + correction from FMRIprep
```
Comment (oesteban) [inline analysis_plan.md]: Similar - what do you plan to do exactly?

**Pull request merged**